region/brinstar/green.json

```diff
@@ -1848,9 +1875,35 @@
               "id": 1,
               "strats": [
                 {
-                  "name": "Base",
+                  "name": "Kill the Beetoms",
                   "notable": false,
-                  "requires": []
+                  "requires": [
+                    {"enemyKill":{
```

Comment:
Do you think this needs a separate strat for a Bombs kill? It looks like you need to take a couple hits.

Reply:
Added some bomb strats.